src/puppetlabs/trapperkeeper/services/webserver/jetty9_core.clj

```diff
@@ -546,6 +546,8 @@
                                maybe-logged)]
       (.setHandler s statistics-handler)
       (.setStopTimeout s (or shutdown-timeout default-graceful-stop-timeout))
+      (when-let [script (:post-config-script options)]
```

Comment:
Well, in that case it sounds like it comes down to "should I break from our 'best practice' now, in order to try be defensive about someone else breaking from our 'best practice' in the future?"  To which I'd lean "no".  :)

So I'd rather leave it here unless you try harder to convince me :)